main/integration-tests/support/setup.js

```diff
@@ -60,11 +60,12 @@ class Setup {
     // aws instance
     this.aws = await initAws({ settings: this.settings });
 
-    // index assigned to default test project provided
-    this.defaultIndexId = await this.getDefaultIndexId();
-
-    // AWS account ID linked to default test project provided
-    this.defaultAwsAccountId = await this.getDefaultAwsAccountId();
+    // An object to abstract out the default setup IDs (eg. test project ID)
+    this.defaults = {};
+    this.defaults.projectId = this.settings.get('projectId');
+    this.defaults.indexId = await this.getDefaultIndexId();
+    this.defaults.awsAccountId = await this.getDefaultAwsAccountId();
+    this.defaults.stepTemplateId = 'st-obtain-write-lock';
```

Comment:
For future enhancement,  we can capture this in a different file similar to how we did the getGenerators() and getServices().

Reply:
Added as a comment, for now.